README.md

```diff
@@ -33,7 +33,7 @@ git submodule update --init --recursive
 
 **Requirements**:
 
-- [Connext DDS 6.1.0](https://community.rti.com/content/page/downloads)
+- [Connext DDS 6.1.2](https://community.rti.com/content/page/downloads)
```

Comment:
Is this still compatible with 6.1.0. If that's the case perhaps you could specify you're compatible with 6.1.x or something along those lines.

Reply:
we have branches with the specific releases. I agree that we should come up with a more generic solution, probably we should do it for the following release